startup/970-load.py

```diff
@@ -58,7 +62,14 @@
 
 # TODO: fix for queueserver - need to exclude interaction:
 # /opt/bluesky/queueserver/queueserver-config.yml - check for existence of this file and load silently...
-beamline_config = _load_beamline_config(glbl['blconfig_path'])
+
+
+print("loading beamline config")
+
+# removing human input for automating queueserver setup by setting test=True
+beamline_config = _load_beamline_config(glbl['blconfig_path'], test=True)
```

Comment:
We should use the check if the worker is active and use the config non-interactively. Otherwise, we can use the interactive mode.

```py
if is_re_worker_active():
    ...
```